### PR TITLE
Add the acceptance test

### PR DESCRIPTION
**Pull request by voscob**
### Description

I've added the acceptance test 🥳

### Quick checks:

- [x] I have followed the [Code Guidelines](https://github.com/ConduitIO/conduit/blob/main/docs/code_guidelines.md).
- [x] There is no other [pull request](https://github.com/conduitio-labs/conduit-connector-clickhouse/pulls) for the same update/change.
- [x] I have written unit tests.
- [x] I have made sure that the PR is of reasonable size and can be easily reviewed.

### CODE REVIEW

File: acceptance_test.go [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+	_, err = db.Exec(fmt.Sprintf("DROP TABLE %s", table))
+	if err != nil {
+		return fmt.Errorf("execute drop table query: %w", err)
+	}
+
+	return nil
+}
+
+// randString generates a random string of length n.
+// (source: https://stackoverflow.com/a/47676287)
```
Comment (maksenius) [inline acceptance_test.go:137]: why we need have link to stackoverflow ?

Comment (voscob) [reply acceptance_test.go:137]: I think it's not important, but I've found the link to the source in the [conduit-connector-sdk](https://github.com/ConduitIO/conduit-connector-sdk/blob/main/acceptance_testing.go#L318).

Comment (maksenius) [reply acceptance_test.go:137]: hm, ok
But it is for you,you can leave it and close comment

**Review by maksenius — APPROVED** [review]

File: acceptance_test.go (hunk 1 above)
Comment (voscob) [reply acceptance_test.go:137]: Got it. 

And you know what? I think it would be better to delete such links.

Thank you!

Comment (voscob) [reply acceptance_test.go]: Removed.

**Pull request merged**